diesel/src/expression/unchecked_bind.rs

```diff
@@ -0,0 +1,57 @@
+use std::marker::PhantomData;
+
+use backend::Backend;
+use query_builder::*;
+use result::QueryResult;
+use types::{ToSql, HasSqlType};
+
+#[derive(Debug, Clone, Copy)]
+#[must_use="Queries are only executed when calling `load`, `get_result` or similar."]
+pub struct UncheckedBind<Query, Value, ST> {
+    query: Query,
+    value: Value,
+    _marker: PhantomData<ST>,
+}
+
+impl<Query, Value, ST> UncheckedBind<Query, Value, ST> {
+    pub fn new(query: Query, value: Value) -> Self {
+        UncheckedBind {
+            query,
+            value,
+            _marker: PhantomData,
```

Comment:
Oh, is this syntax in stable yet? If so, only very recently. This probably bumped us off to "only works with 1.17+".